constellations/gemini-user/src/modals/modal.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+require("./modal.sass");
+import React from "react";
+import srcClose from "../../assets/close_black.svg";
+
+const classnames = require("classnames");
+
+export class Modal extends React.Component {
```

Comment:
do you mind adding docs to this Modal?
Like what props values are available and what they do. And then a simple example of how to use it.

Thanks!!

Reply:
props to list:
content, modalClassName, show, persistent, withClose, onDismiss,